CONTRIBUTING.md

```diff
@@ -41,7 +41,7 @@ If you are adding a new command, create a new test suite (that is a `.bats` file
 
 See how other tests are implemented, and use them as starting point. Use our `envrc_load`
 defined at `test_helpers.bash` in order to load the direnv environment on your test.
-Also, be sure to read about [`bats-core`](https://github.com/bats-core/bats-core) usage, most importantly it's `run` and `$output`.
+Also, be sure to read about [`bats-core`](https://github.com/bats-core/bats-core) usage, most importantly its `run` and `$output`.
```

Comment:
"it's" = "it is", *not* a possessive. English is weird and confusing.

Reply:
me being almost deaf, learned English by reading mailing-lists about parrot-vm :D (lurking on what perl people said between blessings and weird symbols) you can bet it, this one hits me frequently haha.

Thanks for fixing.

Reply:
Ha! No problem.